README.md

```diff
@@ -34,67 +34,56 @@ For a more detailed overview of the repository, please see the documents on the
 
 ## Getting Started
 
+We recommend [conda](https://docs.conda.io/projects/conda/en/latest/glossary.html?highlight=environment#conda-environment) for environment management, and [vscode](https://code.visualstudio.com/) for development. To install the recommenders package and run an example notebook:
+
+```bash
+# Create and activate a new conda environment
+conda create -n <environment_name> python=3.9
+conda activate <environment_name>
+
+# Install the recommenders package with examples
+pip install recommenders[examples]
+
+# create a Jupyter kernel
+python -m ipykernel install --user --name <environment_name> --display-name <kernel_name>
+
+# Clone this repo within vscode or using command:
+git clone https://github.com/microsoft/recommenders.git
+
+# Within vscode:
+#   1. Open a notebook, e.g., examples/00_quick_start/sar_movielens.ipynb;  
+#   2. Select Jupyter kernel <kernel_name>;
+#   3. Run the notebook.
+```
+
+For more information about setup including extras, as well as configurations for GPU, Spark and Docker container, see the [setup guide](SETUP.md).
+<!--
+In addition to the core package, several extras are provided, including:
++ `[examples]`: Needed for running examples.
++ `[gpu]`: Needed for running GPU models.
++ `[spark]`: Needed for running Spark models.
++ `[dev]`: Needed for development for the repo.
++ `[all]`: `[examples]`|`[gpu]`|`[spark]`|`[dev]`
++ `[experimental]`: Models that are not throughly tested and/or may require additional steps in installation).
++ `[nni]`: Needed for running models integrated with [NNI](https://nni.readthedocs.io/en/stable/).
+-->
+<!--
 Please see the [setup guide](SETUP.md) for more details on setting up your machine locally, on a [Data Science Virtual Machine (DSVM)](https://azure.microsoft.com/en-gb/services/virtual-machines/data-science-virtual-machines/) or on [Azure Databricks](SETUP.md#setup-guide-for-azure-databricks).
 
 The installation of the recommenders package has been tested with 
-- Python versions 3.6 - 3.9 and [venv](https://docs.python.org/3/library/venv.html), [virtualenv](https://virtualenv.pypa.io/en/latest/index.html#) or [conda](https://docs.conda.io/projects/conda/en/latest/glossary.html?highlight=environment#conda-environment)
+- Python versions 3.7 - 3.9 and [venv](https://docs.python.org/3/library/venv.html), [virtualenv](https://virtualenv.pypa.io/en/latest/index.html#) or [conda](https://docs.conda.io/projects/conda/en/latest/glossary.html?highlight=environment#conda-environment)
 
 and currently does not support version 3.10 and above. It is recommended to install the package and its dependencies inside a clean environment (such as [conda](https://docs.conda.io/projects/conda/en/latest/glossary.html?highlight=environment#conda-environment), [venv](https://docs.python.org/3/library/venv.html) or [virtualenv](https://virtualenv.pypa.io/en/latest/index.html#)).
+-->
```

Comment:
Kind of same question as before, what do you think of removing this as well. In the new getting started, there is enough information

Reply:
yes will do.